Automattic/wp-calypso
# Performance Profiler: Make CrUX details responsive

Resolves https://github.com/Automattic/dotcom-forge/issues/8911

## Proposed Changes

* In the CrUX details section, wrap the components when there is no free space
* Make the chart responsive to different sizes of screens

> [!NOTE]  
> This PR is based on the changes done in https://github.com/Automattic/wp-calypso/pull/93994

### Recording of changes
https://github.com/user-attachments/assets/263a880c-79d6-4b46-96c5-7e92e6372748


## Why are these changes being made?


* To make responsive the CrUX details section

## Testing Instructions



* Navigate to the Speed Test tool
* Resize the browser window to different sizes
* Make sure the behavior of the CrUX range chart and history chart works as expected. They should wrap for small screens
* Make sure the size of the charts adapt to the screen width.

**Note:** The history chart does not seem to resize correctly when dynamically resizing to small screens. This should be further investigated. 


## Pre-merge Checklist
